src/test/java/net/minecraftforge/test/TestCapabilityMod.java

```diff
@@ -47,7 +47,7 @@ public void preinit(FMLPreInitializationEvent evt)
     @SubscribeEvent
     public void onInteract(PlayerInteractEvent.LeftClickBlock event)
     {
-        if (event.getItemStack() == null) return;
+        if (event.getItemStack().func_190926_b()) return;
```

Comment:
This is not really necessary any more, because below the item retrieved will be air.